test/e2e/tests/identity/account-syncing/importing-private-key-account.spec.ts

```diff
@@ -48,7 +48,7 @@ describe('Account syncing - Import With Private Key', function () {
           });
           const homePage = new HomePage(driver);
           await homePage.check_pageIsLoaded();
-          await homePage.check_expectedBalanceIsDisplayed();
```

Comment:
before, the default value for this method was '0' so we didn't pass any param. Now it's '25' to make it more convenient, as in the majority of cases we do have balance. In this case we don't so, now we need to explicitly pass the '0'